blocks/required-documents/required-documents.css

```diff
@@ -0,0 +1,100 @@
+.block.required-documents .req-type {
+  display: none;
+}
+
+.block.required-documents .req-summary {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.block.required-documents {
+  background-color: white;
+  border-radius: 1.5rem;
+  padding-inline: 2rem;
+
+  /* border: 1px solid var(--lightgrey); */
```

Comment:
please remove unrequired comment

Reply:
@asthabh23 Here, the problem is that in the UE it uses "shift-enter/return" in the rich text, so for it to keep the original padding just use "enter/return" on the different paragraphs. I attached how it looks in both cases. 
<img width="1202" alt="image" src="https://github.com/user-attachments/assets/af2049ef-21cf-49c4-8930-f2a42ead3c4e">

Reply:
yes, I think we can do that, shouldn't be an issue

Reply:
I updated the paddings and the changed the rich text styles so I think it's good now